lisa/testsuite.py

```diff
@@ -584,18 +582,6 @@ def start(
         suite_log = self.__log
         suite_error_stacktrace = None
         #  replace to case's logger temporarily
-        if hasattr(self, "before_suite"):
-            (
-                is_suite_continue,
-                suite_error_message,
-                suite_error_stacktrace,
-            ) = self.__suite_method(
-                self.before_suite,  # type: ignore
-                test_kwargs=test_kwargs,
-                log=suite_log,
-            )
-            self.__print_warning("before_suite", "before_case")
```

Comment:
I think we should add one more step, to raise an error, so it's obvious to people to fix.